commands/narrator/night/others/lynch.js

```diff
@@ -1,99 +0,0 @@
-
-    // check if the first 2 votes are same or if there are not enough votes
-    if (condition && db.get(`game`).gamemode !== "forcevote") {
-        await dayChat.send(`${getEmoji("votingme", client)} The Villagers couldn't decide on who to lynch!`)
-    } else {
-        let guy = db.get(`player_${sortedVotes[0][0]}`) // get the player to lynch
-
-        // check if they are a Handsome Prince or an Idiot
-        if (["Handsome Prince", "Idiot"].includes(guy.role)) {
-            // send the unsuccessful message
-            if (guy.role === "Handsome Prince") {
-                await dayChat.send(`${getEmoji("votingme", client)} The Villagers tried to lynch **${players.indexOf(guy.id) + 1} ${guy.username} (${getEmoji("handsome_prince", client)} Handsome Prince)**, but they were too handsome to be killed today.`)
-            }
-            if (guy.role === "Idiot") {
-                await dayChat.send(`${getEmoji("votingme", client)} The Villagers tried to lynch **${players.indexOf(guy.id) + 1} ${guy.username} (${getEmoji("idiot", client)} Idiot)**, but instead made them lose their ability to vote.`)
-                db.set(`player_${guy.id}.lynched`, true) // set the lynched status to true
-            }
-        } else {
-            let result = await protection(client, guy.id) // get the result of the protection
-
-            // if they were protected
-            if (result === true) {
-                // send the unsuccessful message
-                await dayChat.send(`${getEmoji("votingme", client)} The villagers tried to lynch **${players.indexOf(guy.id) + 1} ${guy.username}**, but they were protected!`)
-            } else {
-                // check if the player is stubborn wolf that has 2 lives
-                let getResult = await stubbornWerewolves(client, guy) // checks if the player is stubborn wolf and has 2 lives
-                if (getResult === true) return false // exits early if the player IS stubborn wolf AND has 2 lives
-                // kill the player normally
-                db.set(`player_${guy.id}.status`, "Dead") // change the status of the player
-                let member = await guild.members.fetch(guy.id) // get the discord member - Object
-                let memberRoles = member.roles.cache.map((r) => (r.name === "Alive" ? "892046207428476989" : r.id)) // get the roles of the discord member
-                let role = guy.role
-                if (guy.tricked) role = "Wolf Trickster"
-                await dayChat.send(`${getEmoji("votingme", client)} The Villagers lynched **${players.indexOf(guy.id) + 1} ${guy.username} (${getEmoji(role.toLowerCase().replace(/\s/g, "_"), client)} ${role})**!`) // send a message to day chat
-                await member.roles.set(memberRoles) // set the roles
-                client.emit("playerKilled", db.get(`player_${guy.id}`), "village")
-            }
-        }
-    }
-
-    db.delete(`game.noVoting`)
-    db.delete(`game.isShadow`)
-}
```

Comment:
what file is doing that now?

Reply:
This was a duplicate file when I merged the bugs pr before this. The other file is at night/kills/lynch.js